addons/website/views/snippets/s_image_gallery.xml

```diff
@@ -12,11 +20,11 @@
 <template id="s_image_gallery_options" inherit_id="website.snippet_options">
     <xpath expr="." position="inside">
         <div data-js="gallery" data-selector=".o_gallery">
-            <we-select string="Mode">
+            <we-select string="Mode" data-dependencies="!slideshow_mode_opt">
                 <we-button data-mode="grid" data-name="grid_mode_opt">Grid</we-button>
                 <we-button data-mode="masonry" data-name="masonry_mode_opt">Masonry</we-button>
                 <we-button data-mode="nomode">Float</we-button>
-                <we-button data-mode="slideshow" data-name="slideshow_mode_opt">Slideshow</we-button>
+                <we-button data-mode="slideshow" data-name="slideshow_mode_opt" class="d-none">Slideshow</we-button>
```

Comment:
Compatibility matters in that discussion. Again I did not check the code, just want to be sure you want this changed for real reasons. I am pretty sure there were lots of scss and options in common during the specs step :wink:

Reply:
@bvr-odoo after discussion with qsm, keep this workaround for now. Just need to remove the "slideshow" method in the js, since you can no longer convert to a slideshow.